pocs/focuser/simulator.py

```diff
@@ -31,39 +54,24 @@ def connect(self):
         self._serial_number = 'SF{:04d}'.format(random.randint(0, 9999))
         self._min_position = 0
         self._max_position = 22200
+        if self.position is None:
+            self._position = random.randint(0, 22200)
```

Comment:
```suggestion
            self._position = random.randint(0, self._max_position)
```

Not that it will matter much here, but still.

Reply:
Fair point.